### PR TITLE
feat: very minimal CSV export of users and maps in admin

### LINKED ISSUE #2020: Add an export button in the admin panel
**Is your feature request related to a problem? Please describe.**
Currently it's tedious for non technical admins to export the list of users / list of maps. 
Same for maps.

**Describe the solution you'd like**
It'd be great to have an export button that downloads a csv when clicked.
In order to avoid downloading the full db, being able to aply filters and export the selection would be interesting to have but not mandatory.

**Describe alternatives you've considered**
I've been copy pasting a paginated list into a table but I'm missing info.
Having a queryable api would be an interesting approach as it would open the data to non admins. 

**Additional context**
I've been doing this for a small instance.

### PR DESCRIPTION
**Pull request by yohanboniface**
fix #2020

### CODE REVIEW

File: umap/admin.py [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
 
 
+class CSVExportMixin:
+    actions = ["as_csv"]
+
+    @admin.action(description=_("CSV Export"))
+    def as_csv(self, request, queryset):
+        response = HttpResponse(
+            content_type="text/csv",
+            headers={"Content-Disposition": 'attachment; filename="export.csv"'},
```
Comment (davidbgk) [inline umap/admin.py:19]: I would but a `csv_filename` on the class to get a comprehensible one (my download folder is a mess 🙈), like `umap_export_users.csv`? Even better with the date of the export?

Comment (yohanboniface) [reply umap/admin.py:19]: Or could I use the model name in lower case ?

File: umap/admin.py [hunk 2]
```diff
@@ -11,10 +37,22 @@ class TileLayerAdmin(admin.ModelAdmin):
     list_editable = ("rank",)
 
 
-class MapAdmin(admin.GISModelAdmin):
+class MapAdmin(CSVExportMixin, admin.GISModelAdmin):
     search_fields = ("name",)
     autocomplete_fields = ("owner", "editors")
     list_filter = ("share_status",)
+    csv_fields = (
```
Comment (virgile-dev) [inline umap/admin.py:44]: Can you add the number of features the map contains @yohanboniface  ?

File: umap/admin.py [hunk 3]
```diff
@@ -30,9 +68,23 @@ class TeamAdmin(admin.ModelAdmin):
     filter_horizontal = ("users",)
 
 
+class UserAdmin(CSVExportMixin, UserAdminBase):
+    csv_fields = [
```
Comment (virgile-dev) [inline umap/admin.py:72]: Can you add a user map count @yohanboniface ?

Comment (virgile-dev) [reply umap/admin.py:72]: Is there anything like last_edit containing the date of the last edit ?

File: umap/admin.py (hunk 2 above)
Comment (yohanboniface) [reply umap/admin.py:44]: Nope, too costly to compute :/

**Pull request merged**